exercises/go-counting/canonical-data.json

```diff
@@ -0,0 +1,170 @@
+{
+  "exercise": "go-counting",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Black corner territory on 5x5 board",
+      "property": "territory",
+      "input": {
+      "board": [
+        "  B  ",
+        " B B ",
+        "B W B",
+        " W W ",
+        "  W  "
+                 ],
```

Comment:
I would align this with the same column that `"board"` is; same with all other closing bracket lines

```
"input": {
  "board": [
    "stuff"
  ],
  "x": 0,
}